src/modules/flow/iio/iio.json

```diff
@@ -945,6 +945,210 @@
+           "name": "iio_device"
+         },
+         {
+           "data_type": "int",
+           "description": "IIO buffer size. If -1, buffering is disabled. If 0, will use default buffer size. If enabled (> 0), a trigger is necessary to perform readings. Set it using 'iio_trigger_name'; if no trigger name is set, it will attempt to create a default one, which is activated via TICK port.",
+           "default": 0,
+           "name": "buffer_size"
+         },
+         {
+           "data_type": "string",
+           "description": "IIO trigger name. Name of IIO trigger that should be associated to this device for buffered readings. If not set and buffer enabled, will try to use device current trigger, if any. If none, will attempt to create a sysfs trigger",
+           "name": "iio_trigger_name",
+           "default": null
+         },
+         {
+           "data_type": "boolean",
+           "default": true,
+           "description": "If should use device own default scale. If false, it will attempt to use scale option.",
+           "name": "use_device_default_scale"
+         },
+         {
+             "data_type": "float",
+             "default": 0,
+             "description": "Scale to applied to device raw readings",
+             "name": "scale"
+         },
+         {
+           "data_type": "boolean",
+           "default": true,
+           "description": "If should use device own default offset. If false, it will attempt to use offset_x, offset_y and offset_z options.",
+           "name": "use_device_default_offset"
+         },
+         {
+             "data_type": "float",
+             "default": 0,
+             "description": "Offset to be added to device raw readings",
+             "name": "offset"
+         },
+         {
+             "data_type": "int",
+             "default": -1,
+             "description": "Sampling frequency of the sensor. If -1, use device default",
+             "name": "sampling_frequency"
+         },
+         {
+             "data_type": "drange-spec",
+             "default": {
+               "max": "DBL_MAX",
+               "min": "-DBL_MAX",
+               "step": "DBL_MIN"
+             },
+             "description": "Range of output packet. Usually, is the output range of used sensor. It'll be the 'min' and 'max' fields of drange packets sent on OUT port.",
+             "name": "out_range"
+         }
+       ],
+       "version": 1
+      },
+      "out_ports": [
+        {
+         "data_type": "float",
+         "description": "Light data read in lux.",
```

Comment:
@edersondisouza some light sensors provide the data in lux, some are raw data. For raw data we will use the scale factor to process them. For "unit-less", do you mean we need to change it to  
"Light data read in lux(unit-lesss)" ?